whylog/constraints/__init__.py

```diff
@@ -191,6 +216,15 @@ def _verify_both(self, group_contents, param_dict):
     def verify(self, group_contents, param_dict):
         pass
 
+    def _validate_params(self):
+        problems = super(TimeConstraint, self)._validate_params()
+        param_min_delta = self.params.get(self.MIN_DELTA)
+        param_max_delta = self.params.get(self.MAX_DELTA)
+        if param_min_delta is not None and param_max_delta is not None:
+            if param_min_delta > param_max_delta:
```

Comment:
Change to `if param_min_delta >= param_max_delta:` If they equal something is wrong here.